src/sonic-yang-models/tests/yang_model_tests/tests_config/vxlan.json

```diff
@@ -0,0 +1,141 @@
+{
+    "VXLAN_VALID_TEST": {
+        "sonic-vlan:sonic-vlan": {
+            "sonic-vlan:VLAN": {
+                "VLAN_LIST": [
+                    {
+                        "name": "Vlan100"
+                    }
+                ]
+            }
+         },
+        "sonic-vxlan:sonic-vxlan": {
+            "sonic-vxlan:VXLAN_TUNNEL": {
+                "VXLAN_TUNNEL_LIST": [
+                    {
+                        "name": "vtep1",
+                        "src_ip": "1.2.3.4"
+                    }
+                ]
+             },
+            "sonic-vxlan:VXLAN_EVPN_NVO": {
+                "VXLAN_EVPN_NVO_LIST": [
+                    {
+                        "name": "nvo1",
+                        "source_vtep": "vtep1"
+                    }
+                ]
+             },
+            "sonic-vxlan:VXLAN_TUNNEL_MAP": {
+                "VXLAN_TUNNEL_MAP_LIST": [
+                    {
+                        "name": "vtep1",
+                        "mapname": "map_100_Vlan100",
+                        "vlan": "Vlan100",
+                        "vni": "100"
+                    }
+                ]
+             }
+         }
+    },
+    "VXLAN_VTEP_WRONG_NAME": {
+        "sonic-vxlan:sonic-vxlan": {
+            "sonic-vxlan:VXLAN_TUNNEL": {
+                "VXLAN_TUNNEL_LIST": [
+                    {
+                        "name": "tunnel1",
+                        "src_ip": "1.2.3.4"
+                    }
+                ]
+             }
+         }
+    },
+    "VXLAN_EVPN_NVO_WITHOUT_VTEP": {
+        "sonic-vxlan:sonic-vxlan": {
+            "sonic-vxlan:VXLAN_EVPN_NVO": {
+                "VXLAN_EVPN_NVO_LIST": [
+                    {
+                        "name": "nvo1",
+                        "source_vtep": "vtep1"
+                    }
+                ]
+             }
+         }
+    },
+    "VXLAN_MAP_WITHOUT_VTEP": {
+        "sonic-vlan:sonic-vlan": {
+            "sonic-vlan:VLAN": {
+                "VLAN_LIST": [
+                    {
+                        "name": "Vlan100"
+                    }
+                ]
+            }
+         },
+        "sonic-vxlan:sonic-vxlan": {
+            "sonic-vxlan:VXLAN_TUNNEL_MAP": {
+                "VXLAN_TUNNEL_MAP_LIST": [
+                    {
+                        "name": "vtep1",
+                        "mapname": "map_100_Vlan100",
+                        "vlan": "Vlan100",
+                        "vni": "100"
+                    }
+                ]
+             }
+         }
+    },
+    "VXLAN_MAP_WITHOUT_VLAN": {
```

Comment:
I don't see corresponding key in tests/vxlan.json. Can you please check and update?

Reply:
This was removed in the last commit as the reference to vlan (import vlan)  was failing the builds.
The intent is that this be re-added later once the import vlan is fixed. Hence it is being removed only in the 
tests and not the tests_config.